xbmc/platform/darwin/ios/IOSScreenManager.mm

```diff
@@ -280,7 +280,10 @@ - (void) screenDisconnect
 //--------------------------------------------------------------
 + (void) updateResolutions
 {
-  CServiceBroker::GetWinSystem()->UpdateResolutions();
+  if (CServiceBroker::GetWinSystem() != nullptr)
+  {
+    CServiceBroker::GetWinSystem()->UpdateResolutions();
+  }
```

Comment:
Because I am unsure if I can set _block on a member of the class (which screensize is) - _block needs to be set to allow the variable to be altered by another run loop in a block.